src/main/java/org/skriptlang/skript/bukkit/loottables/LootContextWrapper.java

```diff
@@ -0,0 +1,57 @@
+package org.skriptlang.skript.bukkit.loottables;
+
+import org.bukkit.Location;
+import org.bukkit.entity.Entity;
+import org.bukkit.entity.HumanEntity;
+import org.bukkit.loot.LootContext;
+import org.jetbrains.annotations.Nullable;
+
+public class LootContextWrapper {
+
+	private final Location location;
```

Comment:
Would it not be best to have this class implement LootContext

Reply:
Unfortunately, LootContext is a final class, it would have to extend the Builder class. I'm not too sure if this would change anything.

Reply:
Ahh didn't realize it's a final class that's sad